LayoutTests/pdf/fullscreen.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<head>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self';">
+</head>
+<body>
+<iframe src="./resources/fullscreen.pdf" width="100%" height="500px"/>
+
+<script>
+    if (window.testRunner) {
+        testRunner.dumpAsText();
+        testRunner.notifyDone();
```

Comment:
You don't need `notifyDone()` if you never called `waitUntilDone()` but does the test need to wait for the PDF to load?

Reply:
Yes, we need to wait for this test case to finish.

I'll need to fix this.

Reply:
Updated the test case to wait for the page to finish loading.